src/webui/www/private/scripts/mocha-init.js

```diff
@@ -387,13 +412,12 @@ initializeWindows = function() {
         if (hashes.length == 1) {
             var hash = hashes[0];
             var row = torrentsTable.rows[hash];
-            if (row) {
-                var name = row.full_data.name;
+            if (row)
```

Comment:
Please keep the curly braces, better safe than sorry.

Reply:
This conditional's code block doesn't seem complex enough to warrant curly braces.